test/support/postgresql/import_examples.rb

```diff
@@ -17,5 +17,83 @@ def should_support_postgresql_import_functionality
         assert_equal 1, result.num_inserts
       end
     end
+
+    describe "importing objects with ids subobjects" do
```

Comment:
Or maybe literally that is what it means - "objects with ids", import their "subobjects"